app/services/ckan/v26/dataset_importer.rb

```diff
@@ -0,0 +1,11 @@
+module CKAN
+  module V26
+    class DatasetImporter
```

Comment:
Maybe this should be `PackageImporter`? We're "importing" the package and "mapping" it to a dataset?

Reply:
I had the same thought. My conclusion was that the files are organised by the local data structures (datasets, inspire_datasets and links); for inspire_datasets there's no equivalent model terminology in CKAN. Perhaps 'DatasetUpdater' would be less ambiguous?

Reply:
+1 for `DatasetUpdater`